sdk/src/loom2d/Textures/RenderTexture.ls

```diff
@@ -0,0 +1,248 @@
+// =================================================================================================
+//
+//	Starling Framework
+//	Copyright 2011-2014 Gamua. All Rights Reserved.
+//
+//	This program is free software. You can redistribute and/or modify it
+//	in accordance with the terms of the accompanying license agreement.
+//
+// =================================================================================================
+
+package loom2d.textures
+{
+    import loom.graphics.Graphics;
+    import loom.graphics.Texture2D;
+    import loom2d.Loom2D;
+    import loom2d.math.Matrix;
+    import loom2d.math.Rectangle;
+    
+    import loom2d.display.BlendMode;
+    import loom2d.display.DisplayObject;
+    import loom2d.display.Image;
+    
+    /** A RenderTexture is a dynamic texture onto which you can draw any display object.
+     * 
+     *  <p>After creating a render texture, just call the <code>drawObject</code> method to render 
+     *  an object directly onto the texture. The object will be drawn onto the texture at its current
+     *  position, adhering its current rotation, scale and alpha properties.</p> 
+     *  
+     *  <p>Drawing is done very efficiently, as it is happening directly in graphics memory. After 
+     *  you have drawn objects onto the texture, the performance will be just like that of a normal 
+     *  texture - no matter how many objects you have drawn.</p>
+     *  
+     *  <p>If you draw lots of objects at once, it is recommended to bundle the drawing calls in 
+     *  a block via the <code>drawBundled</code> method, like shown below. That will speed it up 
+     *  immensely, allowing you to draw hundreds of objects very quickly.</p>
+     *  
+     * 	<pre>
+     *  renderTexture.drawBundled(function():void
+     *  {
+     *     for (var i:int=0; i&lt;numDrawings; ++i)
+     *     {
+     *         image.rotation = (2 &#42; Math.PI / numDrawings) &#42; i;
+     *         renderTexture.draw(image);
+     *     }   
+     *  });
+     *  </pre>
+     *  
+     *  <p>To erase parts of a render texture, you can use any display object like a "rubber" by
+     *  setting its blending mode to "BlendMode.ERASE".</p>
+     * 
+     *  <p>Beware that render textures can't be restored when the Starling's render context is lost.
+     *  </p>
+     *
+     *  <strong>Persistence</strong>
+     *
+     *  <p>Persistent render textures (see the 'persistent' flag in the constructor) are more
```

Comment:
OK cool. I was mostly picking at it to see if it was docs we inherited minus the feature to go with it.